instruments/qubitekk/cc1.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, filelike):
         self.terminator = "\n"
         self._channel_count = 3
         self._firmware = None
+        self._ack_on = False
```

Comment:
on the older versions of the firmware, it will respond to :ACKN OFF with Unknown command